### ISSUE TITLE
Add term ankle dislocation

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10503]
**Preferred term label:**
Ankle dislocation

**Synonyms**
Talo-tibial dislocation

**Definition (free text, please give PubMed ID)**
An abnormal separation between the talo-tibial joint. PMID: 32119497

**Parent term (use hpo.jax.org/app)**
Lower extremity joint dislocation HP:0030311

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Atelosteogenesis type 1 (omim108720), PMID24624349

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl
Adding new term: 
- ID: HP:6001057
- label: Ankle dislocation
- parents: [Lower extremity joint dislocation (HP:0030311)](https://hpo.jax.org/app/browse/term/HP:0030311)
- pmid: PMID:32119497
- def: An abnormal separation between the talo-tibial joint.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10503
- synonyms:
